### DISCUSSION TITLE
Change name of onnxruntime.dll when building

### DISCUSSION THREAD

**Discussion by danyaalk123** [microsoft/onnxruntime#17414] [Other Q&A]
Is it possible to change the name of onnxruntime.dll while building from source so that you dont have any clashes with the one in system32 folder

Copying the right dll to the application folder isnt possible as I am trying to run unit tests using vstest.

**Comment by snnn** [comment]
Sorry, it is not supported.

**Comment by idruker-cerence** [comment]
Suppose you want your binary named cs_onnxruntime. Then you modify the file onnxruntime/cmake/onnxruntime.cmake. See the attached.
[1.zip](https://github.com/microsoft/onnxruntime/files/12527105/1.zip)
